routes/deductions/deductions.spec.js

```diff
@@ -155,9 +104,9 @@ describe('Test /deductions responses', () => {
           const response = await request(app)
             .post(yesNoResponse.url)
             .use(withCSRF(cookie, csrfToken))
-            .send({ [yesNoResponse.key]: 'No', redirect: '/start' })
+            .send({ [yesNoResponse.key]: 'No', redirect: yesNoResponse.noRedir || '/start' })
           expect(response.statusCode).toBe(302)
-          expect(response.headers.location).toEqual('/start')
+          expect(response.headers.location).toEqual(yesNoResponse.noRedir || '/start')
         })
 
         test('it redirects to the checkAnswers when posting "No" and having come from the checkAnswers page', async () => {
```

Comment:
This test will need a refactor due to the branching nature of the /longTermCare/type/cost page. Currently the page redirects to checkAnswers when a no is posted with the ref instead of the /longTermCare/type/amount page.

This will be addressed in https://trello.com/c/qzJUdDs8/516-m-refactor-doyesno-function-to-support-branching-decisions